packages/grafana-perf-tests/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "typescript",
+  "version": "1.0.0",
+  "main": "index.js",
```

Comment:
Does this `index.js` exist in the compiled code?

Reply:
removed